roles/network_plugin/calico/templates/calico.env.j2

```diff
@@ -4,6 +4,11 @@ ETCD_CERT_FILE="{{ calico_cert_dir }}/cert.crt"
 ETCD_KEY_FILE="{{ calico_cert_dir }}/key.pem"
 CALICO_IP="{{ip | default(ansible_default_ipv4.address) }}"
 CALICO_IP6=""
+{% if calico_network is defined and calico_network_backend == 'none' %}
```

Comment:
Ah oops, that should be `calico_network_backend` I'll fix that.  Thanks for taking a look